.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,19 @@
+<!--
+Thanks for submitting a Pull Request!
+
+Please shortly explain your contribution, and if fixing an issue from the tracker, please add a link to the issue.
```

Comment:
Instead of "add a link", it would be better to use the GitHub PR->issue automatic tracking.  Something like:

`if fixing an issue from the tracker, please add "Fixes #XXX", replacing XXX with the issue number`